docs/visualize/timelion-deprecation.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[timelion-deprecation]]
+== Timelion App Deprecation
+
+*With 8.0*, Timelion app is going to be deprecated so there are some actions you should take to copy your existing Timelion worksheets to a dashboard.
+This guide is designed to help you through this process. 
+
+One important note here is that it is going to affect *only the Timelion application*. 
+
+[role="screenshot"]
+image::images/timelion-app.png[]
+{nbsp}
+
+If you use Timelion only as visualization feel free to skip this guide. 
```

Comment:
I think the note above covers this information, so I don't think we need it here.